code/datums/wounds/pierce.dm

```diff
@@ -0,0 +1,175 @@
+
+/*
+	Pierce
+*/
+
+/datum/wound/pierce
+	sound_effect = 'sound/weapons/slice.ogg'
+	processes = TRUE
+	wound_type = WOUND_LIST_PIERCE
+	treatable_by = list(/obj/item/stack/medical/suture)
+	treatable_tool = TOOL_CAUTERY
+	base_treat_time = 3 SECONDS
+	accepts_gauze = TRUE
+	biology_required = list(HAS_FLESH)
+
+	/// How much blood we start losing when this wound is first applied
+	var/initial_flow
+	/// If gauzed, what percent of the internal bleeding actually clots of the total absorption rate
+	var/gauzed_clot_rate
+
+	/// How much flow we've already cauterized
+	var/cauterized
+	/// How much flow we've already sutured
+	var/sutured
+	/// When hit on this bodypart, we have this chance of losing some blood + the incoming damage
+	var/internal_bleeding_chance
+	/// If we let off blood when hit, the max blood lost is this * the incoming damage
+	var/internal_bleeding_coefficient
```

Comment:
Are these default values supposed to be `null` instead of `0`?

Reply:
I define them on each of the actual wound types per their stats, my thought was leaving them undefined tells the reader that it's unique for each subtype